WPMedia/fusion-news-theme-blocks
# Tbrands 95 right rail

## Description

Added a commerce theme for right rail layout.  Specifically, to be used with the product description page.  The design more desktop and mobile can be found here:

https://www.figma.com/file/giQllWgtzGk42KwK2XD9rY/Brand-Browsing-Page-Designs?node-id=1414%3A20894

https://www.figma.com/file/giQllWgtzGk42KwK2XD9rY/Brand-Browsing-Page-Designs?node-id=3589%3A38580 

## Jira Ticket

- [[TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TBRANDS-95)

## Acceptance Criteria

I can select a Right Rail - Arc Layout from the Setup section of Page Builder Editor.
Under the Layout section I see an option for Right Rail - Arc Layout.
Under the Curate View, I see the following sections that I can add blocks to:
Navigation
Body
Full Width (if needed for the Product Recommendation Carousel which is a full body width element)
Footer
The Right Rail - Arc Layout matches the designs for the PDP.
The Right Rail - Arc Layout is responsive and works across all supported device types.

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout tbrands-95-right-rail`
2. Run `npm run storybook`
3. View the `Commerce` story under Right Rail and ensure your theme is set to "commerce" and not "news"


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
